client/app/templates/components/main-landing.hbs

```diff
@@ -21,7 +21,7 @@
       <li class='about__content__list-item--district'>SchoolBot is <a href='http://www2.ed.gov/policy/gen/guid/fpco/ferpa/index.html' target='_blank'>FERPA</a>-compliant, and never transmits or stores identifiable student data.</li>
       <li class='about__content__list-item--district'>We offer SchoolBot as a subscription service, but the code behind it is <a href='https://github.com/Vermonster/schoolbot/blob/master/README.md'>open-source</a> and free for anyone to use or modify. You can contribute new features and integrations, or even run SchoolBot on your own servers.</li>
     </ul>
-    <p>To learn more about SchoolBot and how you can offer it in your district, email us at <a href='mailto:schoolbot@vermonster.com'>schoolbot@vermonster.com</a> and we'll be in touch!</p>
+    <p>To learn more about SchoolBot and how you can offer it in your district, <a href='#contact-form'>contact us</a> and we'll be in touch!</p>
```

Comment:
Above in this file, there is a `#scroll-to` component that could make this jump smoother. It's a drop-in replacement for a link tag.